docs/azureai/azureai-search-document-component.md

```diff
@@ -75,7 +75,22 @@ For more information, see the [Azure AI Search client library for .NET](https://
 
 ## App host usage
 
-[!INCLUDE [azure-component-nuget](../includes/azure-component-nuget.md)]
+To add Azure Cosmos DB hosting support to your <xref:Aspire.Hosting.IDistributedApplicationBuilder>, install the [Aspire.Hosting.Azure.CosmosDB](https://www.nuget.org/packages/Aspire.Hosting.Azure.CosmosDB) NuGet package.
```

Comment:
This is Search, not CosmosDB

Reply:
Good catch, yeah, that was a copy/paste error. I was moving fast, trying to update all the Azure articles, removing an include that discussed `Aspire.Hosting.Azure`, replacing with individual hosting packages. Will fix.